plugins/modules/ecs_service.py

```diff
@@ -557,6 +566,12 @@ def is_matching_service(self, expected, existing):
         if (expected['load_balancers'] or []) != existing['loadBalancers']:
             return False
 
+        if expected['propagate_tags'] != existing['propagateTags']:
+            return False
+
+        if not [x for x in expected['tags'] if x not in existing['tags']]:
```

Comment:
it seems like `existing` does not contains `tags`.
we need to add include tags when pulling from boto3.